qa/tasks/lost_unfound.py

```diff
@@ -141,7 +142,17 @@ def task(ctx, config):
                      pg['state']);
             m = manager.list_pg_unfound(pg['pgid'])
             #log.info('%s' % m)
+
+            # In case you need to compare query output with list_unfound
+            # enable import json above
+            #out = manager.raw_cluster_cmd('pg', pg['pgid'], 'query')
+            #j = json.loads(out)
+            #log.info("pg is %s, query json is %s", pg, j)
+
```

Comment:
no longer needed?

Reply:
It is there in case you are debugging this feature in the future.

Reply:
I removed this and the "import json"